aesara/scan/op.py

```diff
@@ -706,9 +706,7 @@ def tensorConstructor(shape, dtype):
             # output sequence
             o = outputs[idx]
             self.output_types.append(
-                typeConstructor(
-                    shape=(False,) + o.type.broadcastable, dtype=o.type.dtype
-                )
+                typeConstructor((False,) + o.type.broadcastable, o.type.dtype)
```

Comment:
Because the signature hints the `TensorConstructorType` without specifying the arg names.